mod/ckeditor/views/default/elgg/ckeditor/set-basepath.js

```diff
@@ -1,7 +1,8 @@
 /**
  * This view extends elgg.js and sets ckeditor basepath before jquery.ckeditor is loaded
  */
-require(['elgg'], function(elgg) {
+define('elgg/ckeditor/set-basepath', function (require) {
+	var elgg = require('elgg');
```

Comment:
if you do not need require, i like this syntax more...
```js
define(['elgg'], function() {
```

elgg will translate that to
```js
define('elgg/ckeditor/set-basepath', ['elgg'], function() {
```

nonetheless... this LGTM